modules/biscuit/biscuitblaster/main.nf

```diff
@@ -0,0 +1,54 @@
+process BISCUIT_BLASTER {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::biscuit=1.0.2.20220113 bioconda::samblaster=0.1.26 bioconda::samtools=1.15" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-db16f1c237a26ea9245cf9924f858974ff321d6e:17fa66297f088a1bc7560b7b90dc273bf23f2d8c-0':
+        'quay.io/biocontainers/mulled-v2-db16f1c237a26ea9245cf9924f858974ff321d6e:17fa66297f088a1bc7560b7b90dc273bf23f2d8c-0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    path index
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    tuple val(meta), path("*.bai"), emit: bai
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def biscuit_args = task.ext.args ?: ''
+    def samblaster_args = task.ext.args2 ?: ''
+    def sort_args = task.ext.args3 ?: ''
```

Comment:
I would suggest just calling these as `args` `args1` etc. as ths is more consistent with other modules (I guess as a biscuit-module user this would be more readable, for people new to it or other non-biscuit experts looking at the module later it would be better to follow the general trend)